Move to Developer Preview (#51)
xmtp/xmtp-android: README.md

# xmtp-android

![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-yellow)

`xmtp-android` provides a Kotlin implementation of an XMTP message API client for use with Android apps.

Use `xmtp-android` to build with XMTP to send messages between blockchain accounts, including DMs, notifications, announcements, and more.

> This SDK is in **Developer Preview** status and ready for you to experiment with.

Reviewer: An edit based on the definition of Dev Preview - is this OK? =)

```suggestion
This SDK is in **Developer Preview** status and ready for you to start building with.
```

> However, we do **not** recommend using Developer Preview software in production apps. Software in this status is likely to change based on feedback.

Reviewer: An edit based on the definition of Dev Preview - is this OK?

```suggestion
However, we do **not** recommend using Developer Preview software in production apps. Software in this status may change based on feedback.
```

> Specifically, this SDK is working to improve performance and test push notifications.

Reviewer: Thank you for this @nplasterer  - how about something like this? =)

```suggestion
Specifically, we are working on providing push notifications and performance optimizations in the example app.
```

Author: ```suggestion
Specifically, we are working on providing an example of push notification and performance optimizations in the example app.
```

> Specifically, this SDK is working to improve performance and test push notifications.

Author: Push notifications should work with the current sdk but we are working to build out a push example and some performance examples like caching, db, etc... So not sure how to word that 😄
